utils/rpms/daos.spec

```diff
@@ -539,6 +545,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Tue Dec 21 2021 Brian J. Murrell <brian.murrell@intel.com> 2.1.100-17
+- Fix missing R: python*-pyaml and Lmod/lua-lmod
```

Comment:
Where is the fix for Lmod?

Reply:
It went in in an earlier patch.  Updated the changelog.